lib/src/providers/VernaProvider/index.tsx

```diff
@@ -141,22 +132,12 @@ function VernaProvider({
   FieldTemplate,
   Section,
   transformErrors,
-  translations,
 }: PropsWithChildren<VernaProviderProps>) {
-  // Both fullSchema & fullUiSchema are not used by the lib itself but may be
-  // used for further implementation
-  const [fullSchema, setFullSchema] = useState<VernaJSONSchemaType>(defaultSchema);
-  const [fullUiSchema, setFullUiSchema] = useState<UiSchema>(
-    cleanUiSchema(defaultSchema, defaultUiSchema, Section),
-  );
+  const [selector, setSelector] = useState<string | undefined>(defaultSelector);
+  const { schema, ...reducerActions } = useSchemaReducer({ Section, defaultSchema, selector });
```

Comment:
To stick with the jargon of reducer, I think `dispatchers` is a better name.
```suggestion
  const { schema, ...dispatchers } = useSchemaReducer({ Section, defaultSchema, selector });
```